R/indentation_linter.R

```diff
@@ -130,6 +130,8 @@ indentation_linter <- function(indent = 2L, hanging_indent_style = c("tidy", "al
 
   hanging_indent_style <- match.arg(hanging_indent_style)
 
+  global_nodes <- function(nodes) paste0("//", nodes, collapse = "|")
```

Comment:
This should be moved closer to its usage in `glue::glue()`, and nolinted for object_usage since that's a false positive.